ci/kokoro/windows/build-bazel.ps1

```diff
@@ -44,26 +40,6 @@ $common_flags = @("--output_user_root=${bazel_root}")
 
 Write-Host -ForegroundColor Yellow "`n$(Get-Date -Format o) Capture Bazel information for troubleshooting"
 bazel $common_flags version
-bazel $common_flags shutdown
-
-$GOOGLE_CLOUD_CPP_REPOSITORY="google-cloud-cpp"
-$CACHE_BUCKET=if(Test-Path env:GOOGLE_CLOUD_CPP_KOKORO_RESULTS) `
-    {$env:GOOGLE_CLOUD_CPP_KOKORO_RESULTS} else {"cloud-cpp-kokoro-results"}
-$BRANCH_NAME="master"
-$CACHE_FOLDER="${CACHE_BUCKET}/build-cache/${GOOGLE_CLOUD_CPP_REPOSITORY}/${BRANCH_NAME}"
-# Bazel creates many links (aka NTFS JUNCTIONS) and only .tgz files seem to
-# support those well.
-$CACHE_BASENAME="cache-windows-${env:BUILD_NAME}"
-
-$RunningCI = (Test-Path env:RUNNING_CI) -and `
-    ($env:RUNNING_CI -eq "yes")
-$IsCI = (Test-Path env:KOKORO_JOB_TYPE) -and `
-    ($env:KOKORO_JOB_TYPE -eq "CONTINUOUS_INTEGRATION")
-$IsPR = (Test-Path env:KOKORO_JOB_TYPE) -and `
-    ($env:KOKORO_JOB_TYPE -eq "PRESUBMIT_GITHUB")
-$CacheConfigured = (Test-Path env:KOKORO_GFILE_DIR) -and `
-    (Test-Path "${env:KOKORO_GFILE_DIR}/build-results-service-account.json")
-$Has7z = Get-Command "7z" -ErrorAction SilentlyContinue
```

Comment:
This looks like it was just dead code?

Reply:
Yup, it became dead code when we stopped using the tarball cache for these builds.